### PR TITLE
Add a note in the DaemonSet doc for restarting controller manager

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

File: docs/admin/daemons.md [hunk 1]
```diff
@@ -216,6 +216,10 @@ DaemonSet is not enabled by default. Enable it by setting
 achieved by exporting KUBE_ENABLE_DAEMONSETS=true before running kube-up.sh script
 on GCE.
 
+If you intend to restart the api server in an existing cluster to enable this
+feature, you will also need to restart the controller manager
+(`kube-controller-manager`) to ensure that DaemonSet controllers can be created.
+
```
Comment (kelseyhightower) [inline docs/admin/daemons.md]: Can we put a little more details on why this is required? Does the controller manager read specific settings from the API Server at startup?

**Comment by brendandburns** [comment]
LGTM.  Please address Kelsey's comment and then you can feel free to self-lgtm.

File: docs/admin/daemons.md (hunk 1 above)
Comment (yujuhong) [inline docs/admin/daemons.md]: Done.

**Comment by yujuhong** [comment]
PR is outdated since the document has been moved. Closing.

**Pull request closed without merging**